src/integration.test.ts

```diff
@@ -16,7 +16,11 @@ const getKey = (paths: string[], key: string): string => {
 };
 
 describe("Integration Test", (): void => {
-  const EXEC_OPTIONS = { shell: "/usr/bin/bash" };
+  const shellPath =
+    process.platform === "win32"
+      ? "C:\\Program Files\\Git\\bin\\bash.exe"
+      : "/usr/bin/bash";
```

Comment:
Multiple instances: the Jest coverage report complains about these branches in the test files since the coverage report is generated on Linux. In `util.test.ts`, I suggest mocking `process.platform` so that both the Windows and non-Windows flows are exercised in testing. In `integration.test.ts`, it would be sufficient to revert the changes since the existing test should pass regardless of what platform it was run on.